lib/phoenix_token_auth/mailer.ex

```diff
@@ -25,6 +25,7 @@ defmodule PhoenixTokenAuth.Mailer do
   Both config fields have to be functions returning binaries. welcome_email_subject receives the user and
   welcome_email_body the user and confirmation token.
   """
+  def send_welcome_email(user = %{email: email}, confirmation_token) when email == nil do end
```

Comment:
i made it simpler, just a simple conditional if user has email should do.
not really annoyed, but i m trying to make my mindset change from OOP to FP, i thought that the previous way was the correct one ( use functions and match parameters )